en/sensor/sfxx_lidar.md

```diff
@@ -40,6 +40,9 @@ Connect the Lidar the autopilot I2C port as shown below (in this case, for the [
 
 ![SF1XX LIDAR to I2C connection](../../assets/hardware/sensors/sf1xx_i2c.jpg)
 
+:::note
+In case it is the first time you are using the sensor with Pixhawk, you have to enable Pixhawk compatibility in [LightWare Studio](https://lightwarelidar.com/pages/lightware-studio).
```

Comment:
What do you think if we update the message to something a bit more descriptive and helpful, can you tell our users where to find the ENABLE PIXHAWK setting inside the Lightware Studio application?

See my suggestion below:

> Please be aware that Lightware doesn't ship with Pixhawk compatibility enabled by default. To enable support, you have to download LightWare Studio and ENTER INSTRUCTIONS HERE

Reply:
I hadn't heard this, and the doc has been around for a long time - so thanks for providing an update.

As Ramon says, this should have more detailed instructions than just "set it up using lightware". It might be worth having a separate section as a peer of the "Hardware" heading "Lidar Configuration".